mne/viz/topomap.py

```diff
@@ -1066,3 +1067,126 @@ def plot_evoked_topomap(evoked, times=None, ch_type='mag', layout=None,
+        (default) expands to:
+
+        bands = [(0, 4, 'Delta'), (4, 8, 'Theta'), (8, 12, 'Alpha'),
+                 (12, 30, 'Beta'), (30, 45, 'Gamma')]
+
+    vmin : float | callable
+        The value specfying the lower bound of the color range.
+        If None, and vmax is None, -vmax is used. Else np.min(data).
+        If callable, the output equals vmin(data).
+    vmax : float | callable
+        The value specfying the upper bound of the color range.
+        If None, the maximum absolute value is used. If vmin is None,
+        but vmax is not, defaults to np.min(data).
+        If callable, the output equals vmax(data).
+    proj : bool
+        Apply projection.
+    n_fft : int
+        Number of points to use in Welch FFT calculations.
+    picks : array-like of int | None
+        List of channels to use.
+    n_overlap : int
+        The number of points of overlap between blocks.
+    layout : None | Layout
+        Layout instance specifying sensor positions (does not need to
+        be specified for Neuromag data). If possible, the correct layout
+        file is inferred from the data; if no appropriate layout file was
+        found, the layout is automatically generated from the sensor
+        locations.
+    cmap : matplotlib colormap
+        Colormap. For magnetometers and eeg defaults to 'RdBu_r', else
+        'Reds'.
+    agg_fun : callable
+        The function used to aggregate over frequencies. Defaults to np.sum.
+    n_jobs : int
+        Number of jobs to run in parallel.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+    """
+    import matplotlib.pyplot as plt
+    if bands is None:
+        bands = [(0, 4, 'Delta'), (4, 8, 'Theta'), (8, 12, 'Alpha'),
+                 (12, 30, 'Beta'), (30, 45, 'Gamma')]
+
+    if picks is None:
+        picks = pick_types(epochs.info, meg=True, eeg=False,
+                           ref_meg=False)
+
+    if len(picks) == 0:
+        raise RuntimeError('No MEG channels found')
+
+    if layout is None:
+        from ..channels import find_layout
+        pos = find_layout(epochs.info, exclude=[]).pos[picks]
+    else:
+        pos = layout.pos[picks]
+
+    psds, freqs = compute_epochs_psd(epochs, picks=picks, n_fft=n_fft,
+                                     n_overlap=n_overlap, proj=proj,
+                                     n_jobs=n_jobs)
+
+    psds = np.mean(psds, axis=0)
+    for fmin, fmax, title in bands:
+        freq_mask = (fmin < freqs) & (freqs < fmax)
+        if freq_mask.sum() == 0:
+            raise RuntimeError('No frequencies in band "%s" (%s, %s) for '
+                               'sfreq=%s and n_fft %s'
+                               % (title, fmin, fmax,
+                                  epochs.info['sfreq'], n_fft))
+        data = agg_fun(psds[:, freq_mask], axis=1)
+        data = 10 * np.log10(data)
```

Comment:
should be optional. also we should have a normalization option

where all freqs are devided by the sum across the freqs, channel-wise. If you're too lazy, let me add it or open an issue